pages/linux/flatpak-builder.md

```diff
@@ -0,0 +1,24 @@
+# flatpak-builder
+
+> Help build application dependencies.
+> More information: <https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html>.
+
+- Build a Flatpak, then export it to a new repository:
+
+`flatpak-builder {{path/to/build_dir}} {{path/to/manifest}}`
+
+- Build a Flatpak, then export it to the specified repository:
+
+`flatpak-builder --repo={{repo}} {{path/to/build_dir}} {{path/to/manifest}}`
+
+- Build a Flatpak, then install it locally:
+
+`flatpak-builder --install {{path/to/build_dir}} {{path/to/manifest}}`
+
+- Build and sign a Flatpak, then export it to the specified repository:
+
+`flatpak-builder --gpg-sign={{key_id}} --repo={{repository}} {{path/to/manifest}}`
+
+- Run a shell inside of an applications sandbox without installing it:
```

Comment:
```suggestion
- Run a shell inside of an application sandbox, without installing it:
```
I still don't like this sentence.

Reply:
Without the comma sounds good to me.

Reply:
What about a comma here instead

```suggestion
- Run a shell, inside of an application sandbox without installing it:
```

Reply:
That's not great either.  I think entirely without commas is the best version.

Reply:
The first one could've worked I think, though I can't see I'm a big fan of it.
The second one isn't grammatically correct, I don't think. At the very least it's very uncomfortable to read.

I've separately updated "applications" to "application", though.

Reply:
I prefer no comma at all.